rotkehlchen/chain/evm/decoding/gearbox/constants.py

```diff
@@ -0,0 +1,17 @@
+from typing import Final
+
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails
+from rotkehlchen.chain.evm.types import string_to_evm_address
+
+CPT_GEARBOX: Final = 'gearbox'
+GEARBOX_CPT_DETAILS: Final = CounterpartyDetails(
+    identifier=CPT_GEARBOX,
+    label='Gearbox',
+    image='gearbox.png',
```

Comment:
did you add the image? If not add @lukicenturi for how to do it properly since he always re-scales it